specs/deposits.md

```diff
@@ -138,6 +143,11 @@ follows:
 - `context.gas` set to `gasLimit`
 - `context.value` set to `sendValue`
 
+No gas is bought on L2 and no refund is provided. The gas used for the deposit is subtracted from
+the gas pool on L2. Gas usage exactly matches other transaction types (including intrinsic gas).
```

Comment:
Comment from @maurelian "is 'gas pool' a commonly used term? I think 'block gas limit' is more intuitive."

Reply:
(answer from Joshua): It's from geth. Maybe a little used outside of geth.

Reply:
OK, sgtm

Reply:
Was going to comment about this as well.
Could be ".. from the gas pool on L2 (i.e. it reduces the amount of gas that can be consumed by direct L2 transactions)."